### ISSUE TITLE
Translate to inches for people who don't like the symbols

### ISSUE THREAD

**Issue by Floppy** [Floppy/dataslate#567]
It's occurred to me that we could use the replacement code we use to colour the symbols to change them to numerical values instead, as an optional toggle for people who really don't want to use the new measurements. Only gotcha would be some cleverness to change things like `3●` to to `6"` rather than just a straight replacement, which would result in `32"`.

**Comment by lstrzebinczyk** [comment]
There is several features which would require some sort of configuration on a per-roster basis. If you build some basics of how such config would work, other features would be fairly easy to add

**Comment by Floppy** [comment]
Yeah, we should be able to use browser local storage for some sort of preferences thing.
